risc0/zkp/verify/Cargo.toml

```diff
@@ -1,12 +1,23 @@
 [package]
 name = "risc0-zkp-verify"
-version = "0.1.0"
+version = "0.2.0"
 edition = "2021"
 description = "RISC Zero zero-knowledge proof system verify crate"
 license = "Apache-2.0"
 homepage = "https://risczero.com/"
 repository = "https://github.com/risc0/risc0/"
+links = "risc0-zkp-verify"
+
+[features]
+default = ["pure"]
+cxx = ["dep:cxx", "risc0-zkp-core/cxx"]
+pure = ["dep:rand"]
```

Comment:
Can we document what this "pure" feature does?

Reply:
Added some comments